composer.json

```
@@ -29,7 +29,7 @@
         "drupal/devel": "4.1.3",
         "kint-php/kint": "4.1.1",
         "liuggio/fastest": "1.8.0",
-        "mglaman/phpstan-drupal": "0.12.10",
+        "mglaman/phpstan-drupal": "1.1.12",
         "pheromone/phpcs-security-audit": "dev-main",
         "phpcompatibility/php-compatibility": "9.3.5",
         "phpstan/phpstan-deprecation-rules": "0.12.6"
```

We'll need to update this one at the same time.

```suggestion
        "phpstan/phpstan-deprecation-rules": "1.0.0"
```